openssl-sys/src/x509.rs

```diff
@@ -225,6 +225,13 @@ extern "C" {
     pub fn X509_get_pubkey(x: *mut X509) -> *mut EVP_PKEY;
 
     pub fn X509_set_version(x: *mut X509, version: c_long) -> c_int;
+    /// Returns certificate version previously set through `X509_set_version` or the default which
+    /// is `0`.
+    ///
+    /// **Important.** The corresponding OpenSSL method is not&nbsp;expected to fail under its
+    /// correct implementation. Despite that, the method invocation passes through scopes which
+    /// _may_ result in error. Having that it is encouraged to validate returned value.
+    pub fn X509_get_version(x: *const X509) -> c_long;
```

Comment:
There's no need to document function definitions in openssl-sys.

Reply:
Removed redundant doc.